doordeck-sdk-swift/InitializeDoordeck.swift

```diff
@@ -113,6 +113,13 @@ public class Doordeck {
         darkModeActive(darkMode)
     }
     
+    /// This method can be used to puch tile UUID to the SDK to process
```

Comment:
What are you trying to say here? `puch`?

Reply:
push thanks for spotting the mistakes have corrected them